src/Controller/Backend/BackendBase.php

```diff
@@ -37,13 +38,14 @@ public function connect(Application $app)
     /**
      * {@inheritdoc}
      */
-    protected function render($template, array $context = [], array $globals = [])
+    protected function render($template, array $context = [])
     {
         if (!isset($context['context'])) {
             $context = ['context' => $context];
         }
+        $context['context']['bolt_version'] = Version::VERSION;
```

Comment:
Is this needed? I don't think it should be, the constant can be accessed directly from Twig.